src/parser/jobs/rdm/modules/index.tsx

```diff
@@ -0,0 +1,31 @@
+import {AlwaysBeCasting} from 'parser/core/modules/AlwaysBeCasting'
+import {Tincture} from 'parser/core/modules/Tincture'
+import {Combos} from './Combos'
+import {DualCast} from './Dualcast'
+import {Embolden} from './Embolden'
+import {Gauge} from './Gauge'
+import {GeneralCDDowntime} from './GeneralCDDowntime'
+import {Interrupts} from './Interrupts'
+import {MeleeCombos} from './MeleeCombos'
+import {Procs} from './Procs'
+import {Weaving} from './Weaving'
+
+export {DualStatistic} from 'parser/jobs/rdm/statistics/DualStatistic'
```

Comment:
Are there other jobs consuming this component?  If so, we should probably move the component to core instead of having it be exported out of the job in a future update (I realize it's pre-existing and thus out of scope for this update).

Reply:
I do not recall if I was asked to keep it with RDM when I wrote it at the time; this was when some TS was first being added in iirc.  If Ackwell would like it moved to core I'll be happy to do that either in this PR or in a separate one after this one goes through.